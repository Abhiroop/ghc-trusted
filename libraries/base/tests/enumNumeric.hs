main :: IO ()
main = do
    print $ map (/2) ([5..6] :: [Double])
    print $ ([9007199254740990..9007199254740991] :: [Double])
    print $ map (/2) ([9007199254740990..9007199254740991] :: [Double])
    print $ ([9007199254740989..9007199254740990] :: [Double])
    print $ map (/2) ([9007199254740989..9007199254740990] :: [Double])
