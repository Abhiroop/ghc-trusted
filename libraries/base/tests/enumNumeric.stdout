[2.5,3.0]
[9.00719925474099e15,9.007199254740991e15,9.007199254740992e15,9.007199254740992e15]
[4.503599627370495e15,4.5035996273704955e15,4.503599627370496e15,4.503599627370496e15]
[9.007199254740989e15,9.00719925474099e15]
[4.5035996273704945e15,4.503599627370495e15]
